.github/workflows/ci.yml

```diff
@@ -105,30 +102,29 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 18.15
-      # {{SQL CARBON EDIT}} Skip caching for now
-      # - name: Compute node modules cache key
-      #   id: nodeModulesCacheKey
-      #   run: echo "::set-output name=value::$(node build/azure-pipelines/common/computeNodeModulesCacheKey.js)"
-      # - name: Cache node modules
-      #   id: cacheNodeModules
-      #   uses: actions/cache@v3
-      #   with:
-      #     path: "**/node_modules"
-      #     key: ${{ runner.os }}-cacheNodeModules14-${{ steps.nodeModulesCacheKey.outputs.value }}
-      #     restore-keys: ${{ runner.os }}-cacheNodeModules14-
-      # - name: Get yarn cache directory path
-      #   id: yarnCacheDirPath
-      #   if: ${{ steps.cacheNodeModules.outputs.cache-hit != 'true' }}
-      #   run: echo "::set-output name=dir::$(yarn cache dir)"
-      # - name: Cache yarn directory
-      #   if: ${{ steps.cacheNodeModules.outputs.cache-hit != 'true' }}
-      #   uses: actions/cache@v3
-      #   with:
-      #     path: ${{ steps.yarnCacheDirPath.outputs.dir }}
-      #     key: ${{ runner.os }}-yarnCacheDir-${{ steps.nodeModulesCacheKey.outputs.value }}
-      #     restore-keys: ${{ runner.os }}-yarnCacheDir-
+      - name: Compute node modules cache key
+        id: nodeModulesCacheKey
+        run: echo "{value}=$(node build/azure-pipelines/common/computeNodeModulesCacheKey.js)" >> $GITHUB_OUTPUT
+      - name: Cache node modules
+        id: cacheNodeModules
+        uses: actions/cache@v3
+        with:
+          path: "**/node_modules"
+          key: ${{ runner.os }}-cacheNodeModules14-${{ steps.nodeModulesCacheKey.outputs.value }}
+          restore-keys: ${{ runner.os }}-cacheNodeModules14-
+      - name: Get yarn cache directory path
+        id: yarnCacheDirPath
+        if: ${{ steps.cacheNodeModules.outputs.cache-hit != 'true' }}
+        run: echo "{dir}=$(yarn cache dir)" >> $GITHUB_OUTPUT
+      - name: Cache yarn directory
+        if: ${{ steps.cacheNodeModules.outputs.cache-hit != 'true' }}
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.yarnCacheDirPath.outputs.dir }}
+          key: ${{ runner.os }}-yarnCacheDir-${{ steps.nodeModulesCacheKey.outputs.value }}
+          restore-keys: ${{ runner.os }}-yarnCacheDir-
       - name: Execute yarn
-        # if: ${{ steps.cacheNodeModules.outputs.cache-hit != 'true' }} {{SQL CARBON EDIT}} Skip caching for now
+        # if: ${{ steps.cacheNodeModules.outputs.cache-hit != 'true' }}
```

Comment:
Should this be uncommented since the sql carbon edit tag to skip caching was removed?